README.md

```diff
@@ -189,6 +194,102 @@ Number of concurrent runs.
 
 > ⚠️ **`cheap-source-map` options don't work with this plugin**
 
+### `minify`
+
+**webpack.config.js**
+```js
+[
+  new UglifyJsPlugin({
+     minify(file, sourceMap) {
+       const extractedComments = [];
+
+       // Custom logic for extract comments
+
+       const { error, map, code, warnings } = minify(
+         file,
+         { /* Your options for minification */ },
+       );
+
+       return { error, map, code, warnings, extractedComments };
+      }
+  })
+]
+```
+
+By default plugin use `uglify-es` package.
```

Comment:
_uses_?

Reply:
@ooflorent same people want to use own minification (forks) and without this low level function it is impossible, creating new plugin for this situation is not good solution, because it is require copy/paste 99% code of plugin, also we other fix small problems, this will require people to constantly monitor this and cause a lot of inconvenience. Also it is simple way to test alpha/beta/rc/etc versions.